CONTRIBUTING.md

```diff
@@ -22,67 +22,129 @@ Join the TestCafe community on Stack Overflow: ask and answer [questions with th
+When you create a new issue, the template text is automatically added to its body. Complete all the sections in this template to help us understand the problem you are describing. Missing information could delay the processing time.
 
 ## Code Contribution
 
 Follow the steps below when submitting your code.
 
-1. Search the [list of issues](https://github.com/DevExpress/testcafe/issues) to see if there is an issue for the bug or feature you are going to work on or create a new one.
+1. Search the [list of issues](https://github.com/DevExpress/testcafe/issues) to see if there is an issue for the bug or feature you plan to work on or create a new one.
 
-2. If you are going to address an existing issue, check the comment thread to make sure that nobody is working on it at the moment.
+2. To address an already described issue, check the comment thread to make sure that nobody is working on it at the moment. Leave a comment saying that you are willing to fix this issue, and include details on how you plan to do this. Core team members may need to discuss the details of the proposed fix with you. After the green light from them,
+leave a comment saying that you started your work on this issue.
 
-3. Leave a comment saying that you are willing to fix this issue, and if possible, provide details on how you are going to do this.
+3. Install [Node.js](https://nodejs.org/en/), [Google Chrome](https://www.google.com/chrome/) and [Firefox](https://www.mozilla.org/en-US/firefox/new/) on your development machine.
 
-4. Core team members may need to discuss the details of the proposed fix with you. As soon as you get the green light from them,
-  leave a comment saying that you are currently working on this issue.
+4. Fork TestCafe. Clone the fork to your machine and create a new branch. Name this branch with an issue number, for example, `gh852`, `gh853`.
 
-5. Fork TestCafe and create a branch in your fork. Name this branch with an issue number, for example `gh852`, `gh853`.
-  
-    > If you are going to update the documentation follow the steps described in [Contribute to Documentation](#contribute-to-documentation).
+    > To contribute to the docs, follow the [Contribute to Documentation](#contribute-to-documentation) guide.
 
-6. Commit your changes into the branch.
+5. Install dependencies. In the root directory of your local copy, run:
 
-7. Add regression tests to the appropriate sections if you are fixing a bug. You can find these sections by searching for `Regression` in the code.
+    ```sh
+    npm install
+    ```
+
+    or (for [Yarn](https://yarnpkg.com/) users)
+
+    ```sh
+    yarn
+    ```
+
+6. Write code and commit your changes to the branch.
+
+    You can build TestCafe and launch it without running tests:
```

Comment:
That's ambiguous. Do you mean without running *tests that test TestCafe*? (because you do run user's tests from `./tests` in this example).

Reply:
Or... Are *tests that test TestCafe* in `./tests`? Why `without running tests` then?

You see, I'm puzzled (honestly) 😄

Reply:
You're right, that's ambiguous. What about:

```text
 You can build TestCafe and launch it to test changes manually:
```